### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by platers**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (pjkaufman) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]:
```suggestion
**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

Or

Example to reproduce issue with
``` text
example here
```
_Please make sure to include the rule(s) that cause the issue if applicable_

```

Comment (pjkaufman) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: It seems the ending three backticks did not show up in the above suggestion.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```
Comment (pjkaufman) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]:
```suggestion
**Expected behavior**
A clear and concise description of what you expected to happen.

Expected output if applicable:
``` text
expected output here
```
```

Comment (pjkaufman) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: The above also seems to be missing the last 3 backticks.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
Comment (pjkaufman) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: I believe that the only piece of information that will affect most of the tickets will be whether the issue has happened on mobile or on a desktop, but if for some reason we want to keep this information here for the cases where there may be an OS specific problem, that works for me as well.
```suggestion
**Device**
 - [ ] Desktop
 - [ ] Mobile
 
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (pjkaufman) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: I would add the bug label by default here.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (pjkaufman) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: I would add the "rule suggestion" label here by default.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
```
Comment (pjkaufman) [inline .github/ISSUE_TEMPLATE/feature_request.md:15]: Seems that my suggestions are having backticks removed due to the suggestion using backticks:
```suggestion
**Describe the solution you'd like**
A clear and concise description of what you want to happen.

**Please include an example where applicable:**
``` text
original text
```

``` text
expected outcome
```
```

**Comment by pjkaufman** [comment]
I added a couple of suggestions to help make it more specific to the Obsidian Linter plugin, but keep in mind that there are missing ending backticks for code blocks (it seems nesting code blocks in a suggestion does not fare well for Github). Feel free to let me know what you think.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 6 above)
Comment (pjkaufman) [reply .github/ISSUE_TEMPLATE/feature_request.md:15]: The suggestion looks like so:

**Describe the solution you'd like**
A clear and concise description of what you want to happen.

**Please include an example where applicable:**
``` text
original text
```

``` text
expected outcome
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (pjkaufman) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: The above suggestion looks like so:

**Expected behavior**
A clear and concise description of what you expected to happen.

Expected output if applicable:
``` text
expected output here
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (pjkaufman) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: The above suggestion looks like so:

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

Or

Example to reproduce issue with
``` text
example here
```
_Please make sure to include the rule(s) that cause the issue if applicable_

**Comment by platers** [comment]
I agree with all of your suggestions. You can clone this branch and upload your own PR.

**Comment by pjkaufman** [comment]
> I agree with all of your suggestions. You can clone this branch and upload your own PR.

Sounds good. I'll see about getting that done today then.

**Pull request closed without merging**